ui/docker/40-envsubst-on-reearth_config.sh

```
@@ -10,8 +10,8 @@ entrypoint_log() {
 }
 
 auto_envsubst() {
-    local template_file="/tmp/reearth_config.json.tmpl"
-    local output_file="/opt/reearth-flow/reearth_config.json"
+    local template_file="/tmp/reearth_config.json.template"
```

Shellcheck

[warning] In POSIX sh, 'local' is undefined.

(SC3043)